I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 279, in script
    python:
ScriptError: could not find label 'start'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\jkk12\renpy-8.3.3-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\Users\jkk12\renpy-8.3.3-sdk\renpy\main.py", line 652, in main
    run(restart)
  File "C:\Users\jkk12\renpy-8.3.3-sdk\renpy\main.py", line 148, in run
    renpy.execution.run_context(True)
  File "C:\Users\jkk12\renpy-8.3.3-sdk\renpy\execution.py", line 958, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 279, in script
    python:
  File "C:\Users\jkk12\renpy-8.3.3-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'start'.

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.3.24111502
great mediocrity 1.0
Tue Dec 17 14:50:03 2024
